cogs/utils/plainreq.py

```diff
@@ -0,0 +1,10 @@
+"""
+Reducing code complexity (sorta) by wrapping aiohttp GET
+into a simple function.
+"""
+from cogs.common import user_agent
+from aiohttp import ClientSession
+
+
+async def get_req(session: ClientSession, url: str):
+    return await session.get(url, headers=user_agent)
```

Comment:
It is completely unacceptable to have a new file for only 1 function which is itself only a single line. Remove this!
```suggestion
```